skimage/feature/match.py

```diff
@@ -59,7 +59,11 @@ def match_descriptors(descriptors1, descriptors2, metric=None, p=2,
         else:
             metric = 'euclidean'
 
-    distances = cdist(descriptors1, descriptors2, metric=metric, p=p)
+    kwargs = {}
+    # Scipy raises an error if you pass it an extra argument.
```

Comment:
I would rephrase as: "Scipy raises an error if p is directly passed as a keyword argument."

Reply:
ok.